modules/queue/queue_redis.go

```diff
@@ -83,7 +85,7 @@ func NewRedisByteFIFO(config RedisByteFIFOConfiguration) (*RedisByteFIFO, error)
 		queueName: config.QueueName,
 	}
 	fifo.client = nosql.GetManager().GetRedisClient(config.ConnectionString)
-	if err := fifo.client.Ping().Err(); err != nil {
+	if err := fifo.client.Ping(context.Background()).Err(); err != nil {
```

Comment:
Docs [recommend](https://redis.uptrace.dev/changelog/) a global `var ctx = context.TODO()`, maybe it would be more performant to do that instead of recreating that context every time?

Reply:
thank's for the url - was not able to find the cangelog on gh ...